draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -1012,6 +1123,8 @@
       target="RFC7606"/>, a BGP speaker should ignore the Metadata Path
       Attribute if more than one Metadata Path Attribute is within one BGP
       Update message.</t>
+      <!-- XXX JMH - as previously flagged, this is core BGP procedure and
+      should not be repeated in this document. -->
```

Comment:
is it necessary to state that only one Metadata Path Attribute is included in one BGP UPDATE?

Reply:
No.  That's why it should be deleted.

Reply:
Okay. deleted.

Reply:
Removed in recent push.